xbmc/settings/SettingConditions.cpp

```diff
@@ -124,12 +124,17 @@ bool IsMasterUser(const std::string& condition,
   return g_passwordManager.bMasterUser;
 }
 
-bool IsUsingTTFSubtitles(const std::string& condition,
-                         const std::string& value,
-                         const SettingConstPtr& setting,
-                         void* data)
-{
-  return CUtil::IsUsingTTFSubtitles();
+bool HasSubtitlesFontExtensions(const std::string& condition,
+                                const std::string& value,
+                                const SettingConstPtr& setting,
+                                void* data)
+{
+  std::shared_ptr<const CSettingString> settingStr =
```

Comment:
This looks like a good candidate for `auto`

Reply:
done